care.js

```diff
@@ -103,21 +108,83 @@ function doTheTweets() {
 function doTheCodes() {
   var todayCommits = 0;
   var weekCommits = 0;
-
-  var today = spawn('sh ' + __dirname + '/standup-helper.sh', [config.repos], {shell:true});
   todayBox.content = '';
-  today.stdout.on('data', data => {
-    todayCommits = getCommits(`${data}`, todayBox);
-    updateCommitsGraph(todayCommits, weekCommits);
+  weekBox.content = '';
+
+  var showError = function(err) {
+    getCommits("ERROR: "+err, todayBox);
     screen.render();
+  }
+
+  getGitRepos(config.repos, config.reposDepth-1, (err, allRepos) => {
+    if (err) return showError(err);
+    getGitCommits(allRepos, 1, (err, data) => {
+      if (err) return showError(err);
+      todayCommits = getCommits(`${data}`, todayBox);
+      updateCommitsGraph(todayCommits, weekCommits);
+      screen.render();
+    });
+    getGitCommits(allRepos, 7, (err, data) => {
+      if (err) return showError(err);
+      weekCommits = getCommits(`${data}`, weekBox);
+      updateCommitsGraph(todayCommits, weekCommits);
+      screen.render();
+    });
   });
+}
 
-  var week = spawn('sh ' + __dirname + '/standup-helper.sh', ['-d 7', config.repos], {shell:true});
-  weekBox.content = '';
-  week.stdout.on('data', data => {
-    weekCommits = getCommits(`${data}`, weekBox);
-    updateCommitsGraph(todayCommits, weekCommits);
-    screen.render();
+/**
+ * Go through all `repos` and look for subdirectories up to a given `depth`
+ * and look for repositories.
+ * Calls `callback` with array of repositories.
+ */
+function getGitRepos(repos, depth, callback) {
```

Comment:
It looks like PR #36 may land opening up more ES6 usage. It may be good to wait for that to land then refactor these extra functions into isolated modules to keep the main care file smaller.